src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -106,45 +108,32 @@ public SubscriptionConnectionsState OpenSubscription(SubscriptionConnection conn
             return subscriptionState;
         }
 
-        public async Task<long> RecordBatchRevisions(long subscriptionId, string subscriptionName, List<RevisionRecord> list, string previouslyRecordedChangeVector, string lastRecordedChangeVector)
+        public async Task<long> RecordBatchRevisions(SubscriptionConnectionBase.ShardData shardData, long subscriptionId, string subscriptionName, List<RevisionRecord> list, string previouslyRecordedChangeVector, string lastRecordedChangeVector)
```

Comment:
we should remove 'ShardData' from all the methods or rename it appropriately.

Reply:
done